changelogs/fragments/76101-retries-without-until.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
not sure this is 'minor' .. also an entry in the porting guide will probably be needed

Reply:
It's minor by the semver definition because it adds functionality in a backwards-compatible way. None of the existing tests needed to be modified. If you have ideas for situations where the behaviour might have been modified, I can add a test and find out.